src/status_im/chat/events.cljs

```diff
@@ -180,11 +180,13 @@
         (let [chats            (->> all-stored-chats
                                     (map (fn [{:keys [chat-id] :as chat}]
                                            [chat-id (assoc chat :last-message (get-last-stored-message chat-id))]))
-                                    (into {}))]
+                                    (into {}))
+              ;;TODO temporary hide wallet chat, this code should be deleted after wallet contact will be deleted
+              chats' (dissoc chats "wallet")]
```

Comment:
Can we came up with a better variable name?

Reply:
tbh this code is too compact, you can see fast what happens here, and this is temporary, so i believe `temporary-chats-without-wallet` will be extra

Reply:
@flexsurfer Just cursory look so not sure, but I think this dissoc might change conditions for test mocking.

Reply:
fixed

Reply:
why use chat' in the first place ? you can you as-> and have everything in the chat in one threading macro.
also the map function could be moved into a function with a name, and reduce is better suited here than map then into {}

Reply:
i don't want to incorporate this temporary workaround in the code somehow, it must be foreign and easily removable